### DISCUSSION TITLE
'sample linkage' may not always be a GUID

### DISCUSSION THREAD

**Discussion by mslarae13** [microbiomedata/nmdc-schema#2131] [General]
Current description of `slot:sample link`

"A unique identifier to assign parent-child, subsample, or sibling samples. This is relevant when a sample or other material was used to generate the new sample."

Because we no longer required a `source material identifier` or `sample GUID` (GUID is assigned by NMDC) the user may not have a GUID to provide sample links. We should update the description to say "The sample name of samples to be linked to this sample. This should match a sample name provided to NMDC & should identify the sample relationship that is being provided. This is relevant when a sample or other material was used to generate the new sample."

Should update examples. 
core: soil core 1
parent: sample A
pooling: sample 1, sample 2, sample 3

**Comment by bmeluch** [comment]
Double checking - is this edit happening in the submission schema or the NMDC schema?

**Comment by turbomam** [comment]
Please make some valid and invalid examples files for this, no matter which schema you put it in.

Do you need any help putting slots into the submission-schema in a way that bypasses the nmdc-schema?

**Comment by mslarae13** [comment]
nmdc-schema 

Yes, Bea, do you know how to make example files?

**Comment by bmeluch** [comment]
> nmdc-schema

Thanks! Yes I will add example files.

**Comment by mslarae13** [comment]
Blocked:
- How do we structure sample_link?
- What is the use of sample_link?

BACKLOG, will not resolve until post berk
